Source/EasyNetQ/Consumer/ConsumerDispatcher.cs

```diff
@@ -47,8 +52,20 @@ public ConsumerDispatcher(ConnectionConfiguration configuration)
                         logger.ErrorException(string.Empty, exception);
                     }
                 }
+                logger.Info("EasyNetQ consumer dispatch thread finished");
+                }
+                catch (Exception ex)
+                {
+                    logger.Fatal(ex, "EasyNetQ consumer dispatch thread fatal termination");
```

Comment:
Why we need this catch? From the first view, it looks overdefensive.

Reply:
Better safe than sorry.

Reply:
Then, remove it please.

Reply:
Remember of `Thread.Abort()` and other bad things.

Reply:
Done.